README.md

```diff
@@ -36,7 +36,7 @@ this repository will be candidate for a rewrite.
  * [`project`]: String(required): The current project
  * [`health_target`]: String(required): The target of the check. Valid pattern is ${PROTOCOL}:${PORT}${PATH}
  * [`environment`]: String(required): How do you want to call your environment, this is helpful if you have more than 1 VPC.
- * [`backend_security_group`]: String(required): The security group of the ELB backend instances
+ * [`backend_sg`]: List(required): The security groups of the ELB backends instances
```

Comment:
Can you please make this `backend_security_group` again rather than `backend_sg`? With proper tool support, you have syntax completion in your editor. Using acronyms all over the place makes everything much more cryptic. Don't forget that code is read more often than written.

Reply:
+ as it's a list, make it plural :)